jsapp/js/components/processing/sidebar/sidebar.module.scss

```diff
@@ -0,0 +1,16 @@
+@use '../singleProcessingVariables';
+
+.root {
+  height: 100%;
+}
+
+.displays {
+  padding-bottom: singleProcessingVariables.$s-single-processing-tabs;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.displays > :first-child {
+  margin-top: 0 !important;
+}
```

Comment:
I think I used to hack because depending on if it shows translations or not, the first child of `.displays` would change, and depending on which display is first, the `margin-top` would be needed again. This seemed to be less complicated solution